src/dotnetTestExplorer.ts

```diff
@@ -111,21 +111,33 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
         }
     }
 
+    private enableBuild(): string {
+        const dotnetBuildOption = Utility.getConfiguration().get<boolean>("build");
+        return dotnetBuildOption ? "--no-build" : "";

```

Comment:
I have identified that I did these wrong, they're about to be fixed along with some other mistakes I made in a few moments when I update the commit. Sorry about that. Amateur mistakes I made.